utils/utils.jsx

```diff
@@ -196,10 +201,15 @@ export function getChannelURL(state, channel, teamId) {
 }
 
 var canDing = true;
+const notificationSounds = new Map([['Bing', bing], ['Crackle', crackle], ['Down', down], ['Hello', hello], ['Ripple', ripple], ['Upstairs', upstairs]]);
```

Comment:
This might be a bit more readable if it were multiline.

Reply:
Done with this